vyper/evm/opcodes.py

```diff
@@ -84,6 +84,7 @@
     "TIMESTAMP": (0x42, 0, 1, 2),
     "NUMBER": (0x43, 0, 1, 2),
     "DIFFICULTY": (0x44, 0, 1, 2),
+    "PREVRANDAO": (0x44, 0, 1, 2),
```

Comment:
This opcode should probably be updated to the new term

Reply:
I'm not sure tbh about the backwards compatibility - currently, we have two opcodes with the same semantics `DIFFICULTY` & `PREVRANDAO` to keep everything compatible. You would suggest renaming `DIFFICULTY` to `PREVRANDAO` directly, I assume?